lib/EnzymeCore/Project.toml

```diff
@@ -7,11 +7,14 @@ version = "0.7.1"
 Adapt = "3, 4"
 julia = "1.6"
 
+[deps]
+Adapt = "79e6a3ab-5dfb-504d-930d-738a2a938a0e"
```

Comment:
To confirm, this doesn't add the dep on 1.9+?

Reply:
No, on Julia >= 1.9 it will still be only a weak dependency.